articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+
+Deep learning uses algorithms whose functionality is like that of machine learning. One major distinction on each layer is that it has a different interpretation of data. The [neural networks](https://news.mit.edu/2017/explained-neural-networks-deep-learning-0414) learn where data is fetched.
+
+The neurons in the network undertake two tasks. The first includes understanding which feature is optimal for performing a specific task. The second one involves classifying data after many iterations of the learning process. In case there is a mistake, the deep learning model's accuracy increases. Deep learning is, thus, a more complex and more capable version of machine learning.
+
+### Machine Learning vs. Deep Learning
+
+Deep learning differs from machine learning in some aspects. One major attribute is that deep learning learns from its mistakes. In most cases, this is achieved without human intervention. As for machine learning, there is a need for a human to correct the mistake made.
+
+Deep learning then adopts an algorithm then learns not to make that mistake again in the future. This enhances the [multi-level neural networks](https://towardsdatascience.com/multi-layer-neural-networks-with-sigmoid-function-deep-learning-for-rookies-2-bf464f09eb7f?gi=e7b1f9430ac5) to learn from their mistakes. But, this model does not guarantee success or no mistakes in the future. This because a wrong prediction can be made depending on the quality of the data fed to them. The accuracy is dependent on the quality of data used to learn.
+
+### Applications and Conclusion
+
+Deep learning is currently being used in various industries and companies. Most of these companies involve the use of the high tech industry in their operations. Besides this, we, as consumers, are currently using these technologies and applications. People have been using and interact with these innovations without even realizing it.
+
+Google applications like Gmail, Google Search, and Google Maps rely on deep learning. This is because such applications need accurate data to improve their intuitiveness. For instance, Gmail can detect important mails for the user. This is determined by the frequency through which a user gets to communicate. This is made effective by using a smart reply that offers a fast response to any email received. This feature depends on the content of the email received.
+
+Besides this, Google Search uses a prediction that anticipates what one is doing. This includes a prediction through past searches, activities, and places visited. The Google Maps application keeps data that can trace the history of past locations. Due to this, the application makes correct and accurate predictions.
+
+Machine learning stimulates the smooth transition of smart applications today. Most of these applications fetch a customized user experience that improves predictability. As a result, the user can enjoy the fast use of services offered by these applications.
```

Comment:
Personalized services are a result of massive data collection and data crunching. You might want to include this point and build on it as well.

Reply:
I don't think this point was taken into consideration.